test/units/TEST-75-RESOLVED.sh

```diff
@@ -1084,6 +1084,29 @@ testcase_13_varlink_subscribe_dns_configuration() {
         <(jq -cr --seq  '.configuration[] | select(.ifname == "dns0" and .servers != null and .searchDomains != null) | {"dns0":{servers: [.servers[] | .address], domains: [.searchDomains[] | .name]}}' "$tmpfile")
 }
 
+# Test RefuseRecordTypes
+testcase_14_refuse_record_types() {
+    mkdir -p /run/systemd/resolved.conf.d
+    {
+        echo "[Resolve]"
+        echo "RefuseRecordTypes=AAAA SRV TXT"
+    } >/run/systemd/resolved.conf.d/refuserecords.conf
+    ln -svf /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf
```

Comment:
Please save the existing `/etc/resolv.conf`, and restore it on cleanup:
```sh
if [[ -e /etc/resolv.conf ]]; then
    mv /etc/resolv.conf /etc/resolv.conf.bak
fi
ln -s /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf
```

Reply:
done